doc/lua_api.md

```diff
@@ -7562,6 +7562,20 @@ child will follow movement and rotation of that bone.
     * only properties listed in the table will be changed
     * see the 'Object properties' section for details
 * `get_properties()`: returns a table of all object properties
+* `set_observers(observers)`: sets observers (players this object is sent to)
+    * If `observers` is `nil`, the object's observers are "unmanaged":
+      The object is sent to all players (in proximity). This is the default.
+    * `observers` is a "set" of player names: `{[player name] = true, [other player name] = true, ...}`
+        * A player is an observer if and only if the player's name is a key in the `observers` table.
+        * The values are irrelevant. To prevent the mistake of expecting `[player name] = false`
+          to "uninstall" a player as an observer, they must be truthy.
```

Comment:
I would simply say that they must be true, don't want to give modders too much leeway.
(Unrelated to what the engine code actually does)